frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -219,9 +222,31 @@ func (m *CVDToolInstanceManager) launchCVD(req apiv1.CreateCVDRequest, op apiv1.
 	result = OperationResult{Value: cvd}
 }
 
+const (
+	defaultBranch = "aosp-master"
+	defaultTarget = "aosp_cf_x86_64_phone-userdebug"
+)
+
 func (m *CVDToolInstanceManager) launchFromAndroidCI(
 	req apiv1.CreateCVDRequest, op apiv1.Operation) (*apiv1.CVD, error) {
-	artifactsDir, err := m.fetchCVDHandler.Fetch(req.CVD.BuildSource.AndroidCIBuild)
+	build := req.CVD.BuildSource.AndroidCIBuild
```

Comment:
nit: could this be `req.BuildID()`?

That would decouple the internals of the req object from where it's used.

I'm also thinking about supporting multiple build sources: In that scenario the right property to read from the request is probably not `AndroidCIBuild`. That actually means it's probably a better idea to do `buildAPI.GetBuildIDFromRequest(req)`, which would also make `defaultBranch` and `defaultTarget` be properties of the build API too.

Reply:
I think it's fine using the full struct path vs having shortcut methods for accessing specific properties. 

We already support two different build sources, this method is  the one called when the AndroidCIBuild build source is set.